test/run_test.py

```diff
@@ -958,6 +980,18 @@ def exclude_tests(exclude_list, selected_tests, exclude_message=None):
     return selected_tests
 
 
+def must_serial(file: str) -> bool:
+    return (
+        "distributed" in os.getenv("TEST_CONFIG", "") or
+        "functorch" in os.getenv("TEST_CONFIG", "") or
```

Comment:
Minor observation: I did remove functorch from this in the first iterator of this change, and it works fine. In other words, functorch tests can be run in parallel and functorch/test_ops already has its own custom handler